agency/usr/emiso_engine/webserver/image/image.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023 Jean-Pierre Miceli <jean-pierre.miceli@heig-vd.ch>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#include "image.hpp"
+
+namespace emiso {
+
+    namespace image {
+
+        ImageApi::ImageApi(httpserver::webserver *server)
+            : _server(server) {
+                std::string path  = "/images";
+
+            // Create routes and handlers
+            _listHandler = new ListHandler();
```

Comment:
Never freed

Reply:
correct - it is always needed during app live